scripts/update/0.5-0.6/update.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+set -euo pipefail
+
+script_dir=$(cd "$(dirname "${BASH_SOURCE[0]}")" &>/dev/null && pwd -P)
+
+kubectl create ns update-host-files --dry-run=client -o yaml | kubectl apply -f -
```

Comment:
Why do we need `-o yaml` if we are not saving it in a file. Shouldn't just `kubectl create ns update-host-files` be enough?

Reply:
Also as the namespace name is used in multiple occurrences, how about we store it as a variable?

Reply:
> Why do we need `-o yaml` and then pipe the output to `apply` if we are not saving it in a file. Shouldn't just `kubectl create ns update-host-files` be enough? 

This is because `k create` fails if namespace already exists. Doing a yaml output and apply works every time.